store/local_cache_supplier_schemes.go

```diff
@@ -19,7 +19,7 @@ func (s *LocalCacheSupplier) handleClusterInvalidateScheme(msg *model.ClusterMes
 
 func (s *LocalCacheSupplier) SchemeSave(ctx context.Context, scheme *model.Scheme, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
 	if len(scheme.Id) != 0 {
-		defer s.doInvalidateCacheCluster(s.schemeCache, scheme.Id)
+		s.doInvalidateCacheCluster(s.schemeCache, scheme.Id)
```

Comment:
The rest of the changes in this PR amount to "save, then notify", but here we "notify, then save". I guess we're trying to avoid invalidating newly created schemes? Should we keep track up front if this was new and then still notify after?

Actually, this seems like a good case for just preserving the `defer`. 1/5, but I think it's cleaner if you just change them all /to/ using `defer`.

Reply:
Good catch of another case where I removed the defer and added the problem this PR was meant to fix. I'll revert this as well as any other cases where I changed it away from a defer.

I may not change them all to use defer if that complicates the logic though. I'd prefer to keep defers to only one-liners if possible